onnxruntime/core/mlas/lib/power/QuantizePowerVSX.cpp

```diff
@@ -0,0 +1,130 @@
+#include "mlasi.h"
+#include <altivec.h>
+
+template <typename OutputType>
+void
+MLASCALL
+MlasQuantizeLinearVSX(
+    const float* Input,
+    OutputType* Output,
+    size_t N,
+    float Scale,
+    OutputType ZeroPoint
+    )
+{
+    (void) Scale;
```

Comment:
nit:redundant

Reply:
@yufenglee Thanks! It's a workaround to avoid bad GCC warnings with some VSX instruction. I added a note in the code to explain that.